test/e2e/po_admission_webhook_test.go

```diff
@@ -53,7 +54,7 @@ func newSinglePrometheusRule(t *testing.T, name, expr string) *monv1.PrometheusR
 				Rules: []monv1.Rule{{
 					Alert: "alert name",
 					Expr:  intstr.FromString(expr),
-					For:   "15m",
+					For:   &durationFifteenMinutes,
```

Comment:
suggestion for a follow-up PR
```suggestion
					For:   ptr.To(monv1.Duration("15m")),
```

Reply:
fix in next pr